fw/vhost.c

```diff
@@ -2733,56 +2801,66 @@ static TfwCfgSpec tfw_global_frang_specs[] = {
 	{
 		.name = "http_uri_len",
 		.deflt = "0",
-		.handler = tfw_cfgop_frang_uri_len,
+		.handler = tfw_cfgop_frang_glob_set_int,
+		.dest = &tfw_frang_vhost_reconfig.http_uri_len,
 		.allow_reconfig = true,
 	},
 	{
 		.name = "http_body_len",
 		.deflt = "1073741824", /* 1 Gb. */
-		.handler = tfw_cfgop_frang_body_len,
+		.handler = tfw_cfgop_frang_glob_set_long,
+		.dest = &tfw_frang_vhost_reconfig.http_body_len,
 		.allow_reconfig = true,
 	},
 	{
 		.name = "http_strict_host_checking",
 		.deflt = "true",
-		.handler = tfw_cfgop_frang_strict_host_checking,
+		.handler = tfw_cfgop_frang_glob_set_bool,
```

Comment:
We can't use tfw_cfgop_frang_glob_set_bool here because default value is true.
frang_limits {
    http_strict_host_checking false;
 }
 frang_limits {
 }

We have http_strict_host_checking true, but should have false